src/sailing_robot/scripts/sensor_processed_wind_direction

```diff
@@ -11,7 +11,7 @@ class Wind_direction_average():
     def __init__(self):
```

Comment:
Likewise, I think we should reserve `sensor_` for nodes that read data directly from sensors.

I'd call this one something like `processing_smoothed_wind_direction`.